src/useIsShake.ts

```diff
@@ -1,6 +1,7 @@
 import { useState, useEffect } from 'react';
 import { Accelerometer } from 'expo-sensors';
-import { Subscription } from 'expo-sensors/build/Pedometer';
+import { Subscription, isAvailableAsync, requestPermissionsAsync } from 'expo-sensors/build/Pedometer';
+import * as Linking from 'expo-linking';
```

Comment:
This `*` brings in every method from `expo-linking`, but it looks like you are only using `openSettings`. Importing a whole library (could be MBs worth of unused code) when you don't need it can be disastrous for performance. 
```suggestion
import { openSettings } from 'expo-linking';
```